startall.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+## This script will launch the main components for data-api (config, eureka, key, filedatabase)
+## Please run the following components in debug mode: res, data-edge, htsget
```

Comment:
You mean, "Please run the following components manually, in debug mode, from your IDE"?
Yes, that's one possibility. Or another (slightly more convoluted) way, is to start those 3 projects here (from this script), using that magic jdwp string and then people can attach a remote debugger from their IDE.
Pros: all the apps are started from this central bash script.
Cons: setting up 3 remote debug sessions from the IDE is more difficult than starting them manually, in debug mode, from the IDE.